staging/script.xsqueeze/resources/lib/classes/NowPlayingWindow.py

```diff
@@ -295,7 +302,41 @@ def update(self):
         self.updatePlaylistDetails()
         self.updateCoverArtFromURLs()
 
-  ##############################################################################
+        #trigger a hdmi command, if a controller has changed the power status or the volume
+        if constants.TURNONAVR == 'true' and constants.CECSUPPORT == 'true':
+          powerstate = self.player.getpowerstate()
+          if powerstate == 0 and constants.HDMIPOWER == "true":
+            if constants.TURNOFFTV == "true":
+               tv.standby()
+               pass
+            if constants.TURNONAVR == "true":
+               avr.standby()
+            constants.HDMIPOWER = "false"
```

Comment:
constants are supposed to be just that, constant - they are not used to store state